find.py

```diff
@@ -0,0 +1,21 @@
+import os
+from imports import *
+
+def find(field,search):
+    if field=="isbn":
+        list=db.session.query(Book).filter(Book.isbn.like(search))
+        for x in list:
+            return render_template('search.html',list=list)
```

Comment:
- Impliment only the logic(python code) in find.py Dont use render/redirect here. With result from find.py, make the rendering/redirecting in the application.py

- Dont repeat this block of code, many times. Rather use this block of code at the bottom to reduce the code redundancy